docker/all-in-one/Dockerfile

```
@@ -1,4 +1,4 @@
-ARG postgres_version=15.1.0.83
+ARG postgres_version=15.1.0.136
```

```suggestion
ARG postgres_version=15.1.0.148
```

Maybe we could bump into latest? Anyway, not a blocker since everything runs again.

---

Good point - better to have an up-to-date default base